app/scripts/utils/path.js

```diff
@@ -1,26 +1,28 @@
-function Path (protocol, name, path) {
-  this.protocol = protocol
-  this.name = name
-  this.path = path
+export default class Path {
```

Comment:
I'm guessing this is all how new classes are defined in ES6. Otherwise, all of it seems odd.